.github/workflows/gradle.yml

```diff
@@ -69,13 +76,47 @@ jobs:
     - name: Build with Gradle with local and optional remote DocumentDB tests
       run: |
         ./gradlew build -PrunRemoteIntegrationTests=$RUN_REMOTE_INTEGRATION_TESTS
+    - name: Extract gpg-key-pair into file
+      # if: ${{ github.event.inputs.mavenFilesPreparation }}
+      run: |
+        echo "$GPG_KEY_SECRET_KEY_RING" > secring.kbx.b64
+        base64 -d secring.kbx.b64 > secring.kbx
+    - name: Prepare to publish to Maven
+      # if: ${{ github.event.inputs.mavenFilesPreparation }}
+      run: |
+        ./gradlew build publishToMavenLocal -Psigning.secretKeyRingFile=secring.kbx -Psigning.password=$GPG_KEY_PASSWORD -Psigning.keyId=$GPG_SIGN_KEY
     - name: Copy Build
       run: |
         mkdir output
         mkdir output/jarfile
         mkdir output/reports
         cp build/libs/*.jar output/jarfile
         cp -R build/reports output/reports
+    - name: Copy Files for Maven
+      # if: ${{ github.event.inputs.mavenFilesPreparation }}
```

Comment:
Just checking - we do want to only run this step when preparing maven files, correct?

Reply:
I just commented to test in the PR run. I will uncomment soon